src/runner.ts

```diff
@@ -141,7 +164,7 @@ export class GitHubRunners extends Construct {
   private readonly webhook: GithubWebhookHandler;
   private readonly orchestrator: stepfunctions.StateMachine;
   private readonly setupUrl: string;
-  private readonly extraLambdaEnv: {[p: string]: string} = {};
+  private readonly extraLambdaEnv: { [p: string]: string } = {};
```

Comment:
I'm very curious what's this doing here. It's not a big deal as the auto actions will run lint. But do you know why it gives you this result and another for me?

Reply:
Hmm figured out that I was not using prettier 😮 
But I have used prettier now and formatted it again ...

Reply:
Seems not to fix it. Do you use vscode and can setup a default formatter ?

Reply:
I`ve seen that projen has an option to enable prettier and its options. https://projen.io/api/API.html
Can we set this to true ?

<img width="982" alt="image" src="https://user-images.githubusercontent.com/5619511/204162327-bddd76c1-53d4-4074-b679-9b88ff19635d.png">

Reply:
I don't use VScode, but projen did setup eslint. I was under the impression it has very specific rules for things like that and that it's enforced in GitHub workflows.

Reply:
Yeah let's enable prettier in a separate PR.

Reply:
According to eslint documentation we are supposed to already have that:
https://github.com/CloudSnorkel/cdk-github-runners/blob/2aae0e6286cda81fb809aa3e02287767e29a6d28/.eslintrc.json#L83-L86
I'm not sure why it's not enforcing it yet.